Client/package.json

```diff
@@ -25,6 +25,7 @@
     "dayjs": "1.11.11",
     "joi": "17.13.1",
     "jwt-decode": "^4.0.0",
+    "prettier": "^3.3.3",
```

Comment:
Prettier should probably be a dev dependency instead of a build dependency, we don't need it to be included in the build I don't think?

Reply:
Agreed, Alex! I corrected it. Thanks for that!